daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/AT.dfdl.xsd

```diff
@@ -116,8 +116,10 @@
                   <!-- this have BEFORE timing, so for the first local_name 
                     $tns:counter = 1 -->
                     <!-- NOTE: THIS REQUIRES dfdl:newVariableInstance BECAUSE VARIABLES ARE SINGLE ASSIGNMENT IN DFDL -->
-                    <!-- Updating test to use newVariableInstance but the feature is not yet implemented 
-                         so this will need to be reviewed -Beth 8/29/13 -->
+                    <!-- Using newVariableInstance here gives SDE:
+                         newVariableInstance may only be used on group
+                         reference, sequence or choice so this will
+                         need to be reviewed (5/18/20) -->
                   <dfdl:newVariableInstance ref="tns:counter" defaultValue="{ $tns:counter + 1 }" />
```

Comment:
This whole test and example can only be a negative test if it's trying to use a variable as an accumulator, and trying to use newVariableInstance on an element. 

These tests with the 2-letter names like AT are really old (like 10 years now), and concepts in DFDL shifted out from under them. 

I bet we can just drop test AT without losing any coverage.

Reply:
Agreed. I took a look at this and on top of not adding any obviously new coverage, they needed more work than just putting the newVariableInstance call in the right location. I was getting a few other SDE's for other issues too.